plasmapy/formulary/collisions.py

```diff
@@ -1017,7 +1019,8 @@ def Coulomb_cross_section(impact_param: u.m) -> u.m ** 2:
 
     References
     ----------
-    .. [1] https://en.wikipedia.org/wiki/Cross_section_(physics)#Collision_among_gas_particles
+    .. [1] `Cross Section: Collision among gas particles
+       <https://en.wikipedia.org/wiki/Cross_section_(physics)#Collision_among_gas_particles>`__
```

Comment:
Speaking of fragile links, this one should be a permalink:

```suggestion
    .. [1] `Cross Section: Collision among gas particles
       <https://en.wikipedia.org/w/index.php?title=Cross_section_(physics)&oldid=1037954726#Collision_among_gas_particles>`__
```